packages/extension/src/newtab/ShortcutLinks.spec.tsx

```diff
@@ -94,7 +94,6 @@ const defaultSettings: RemoteSettings = {
     'http://custom2.com',
     'http://custom3.com',
     'http://custom4.com',
-    'http://custom5.com',
```

Comment:
Had to reduce the amount to make it easier to debug since the count for placeholder and this list is the same. Also why I had to reduce the count of the tests below by 1.